### PR TITLE
Category page

### PR DESCRIPTION
**Pull request by ShrutiLakhani**
Please review

### CODE REVIEW

**Review by kushalsourav — APPROVED** [review]
looks good for me

**Comment by shruthipatil11** [comment]
looks great

File: Category.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+    <link rel="stylesheet" href="https://canvas-component-lib.netlify.app/Component.css">
+    <title>Document|Category</title>
+</head>
+<body>
+    <div class="desktop-container">
+        <h1>Style Knockout</h1>
+        <div class="Wishlist-container">
+       
+            <!-- Card -->
+            <div class="card2-container wishlist-card">
```
Comment (SinchanaMS) [inline Category.html]: A suggestion - Change the class name to something more generic, the wishlist class here doesn't seem fit.
The rest looks good!

**Review by SinchanaMS — APPROVED** [review]

File: Category.css [hunk 2]
```diff
@@ -0,0 +1,138 @@
...
+    display: flex;
+    flex-direction: column;
+    align-items: center;
+    margin-top: 2rem;
+    height: 26rem;
+  }
+  .Wishlist-container .card2-container,
+  h1,
+  h2,
+  p {
```
Comment (ashishkatiyar2404) [inline Category.css]: avoid tag styling.

**Review by ashishkatiyar2404 — APPROVED** [review]
suggested one change. other looks good to merge
UI looks really good. 
you can add padding-top to images in your both brands

**Pull request merged**